### ISSUE TITLE
Feature: React component for command line input

### ISSUE THREAD

**Issue by clrcrl** [fishtown-analytics/docs.getdbt.com#31]
Would be cool to have a component that looks like a command line prompt. Just to differentiate between "this is code you write in your project" and "this is a command you run"

**Comment by VersusFacit** [comment]
Isn't the dollar sign the thing that signals that? Do we need additional differentiation?

**Comment by clrcrl** [comment]
Yeah absolutely — remember that not everyone is a former unix statems programmer.

By having a react component we can also:
- in the future, render things differently for a cloud-like vs CLI-like interfact
- make the command copyable (without the `$`)

**Comment by VersusFacit** [comment]
Makes sense! So, judging from a quick Google and your comment, we'll need someone with React chops to assist us on creating this feature, right?
